test/parallel/test-zlib-destroy.js

```diff
@@ -13,9 +13,7 @@ const zlib = require('zlib');
   ts.destroy();
   assert.strictEqual(ts._handle, null);
 
-  ts.on('close', common.mustCall(() => {
-    ts.close(common.mustCall());
-  }));
+  ts.on('close', common.mustCall());
```

Comment:
`finished` does not invoke callback on closes stream, this was a semver-major fix does not land on 13